act-rules-format.bs

```diff
@@ -775,37 +773,28 @@ Examples in this section include:
   "hasPart": "dct:hasPart",
   "source": "dct:source",
 
-  "Agent": "foaf:term_Agent",
-  "Person": "foaf:term_Person",
-  "Organization": "foaf:term_Organization",
-  "Group": "foaf:term_Group",
-  "Document": "foaf:term_Document",
-  "name": "foaf:term_name",
-  "firstName": "foaf:term_firstName",
-  "surname": "foaf:term_surname",
-  "mbox": "foaf:term_mbox",
-  "mbox_sha1sum": "foaf:term_mbox_sha1sum",
-  "member": "foaf:term_member",
-  "homepage": "foaf:term_homepage",
-
-  "assertedBy": { "@id": "@type" },
-  "subject": {
-    "@id": "earl:subject",
-    "@type": "schema:WebPage"
-  },
-  "test": {
-    "@id": "earl:test",
-    "@type": "earl:TestCase"
-  },
-  "result": { "@id": "@type" },
-  "mode": { "@id": "@type" },
-  "outcome": { "@id": "@type" },
+  "Agent": "foaf:Agent",
+  "Person": "foaf:Person",
+  "Organization": "foaf:Organization",
+  "Group": "foaf:Group",
+  "Document": "foaf:Document",
+  "name": "foaf:name",
+  "firstName": "foaf:firstName",
+  "surname": "foaf:surname",
+  "mbox": "foaf:mbox",
+  "mbox_sha1sum": "foaf:mbox_sha1sum",
+  "member": "foaf:member",
+  "homepage": "foaf:homepage",
+
+  "assertedBy": { "@type": "@id" },
+  "subject": { "@type": "@id" },
+  "test": { "@type": "@id" },
+  "mode": { "@type": "@id" },
+  "outcome": { "@type": "@id" },
   "pointer": {
     "@id": "earl:pointer",
     "@type": "ptr:CSSSelectorPointer"
-  },
-  "passed": { "@id": "@type" },
-  "failed": { "@id": "@type" }
```

Comment:
passed / failed aren't properties, they are instances. They can only ever be used as a value, so they don't need default type.